pythran/backend.py

```diff
@@ -914,7 +914,7 @@ def visit_Num(self, node):
 
     def visit_Str(self, node):
         quoted = node.s.replace('"', '\\"').replace('\n', '\\n"\n"')
-        return '"' + quoted + '"'
+        return 'pythonic::types::str("' + quoted + '")'
```

Comment:
error C2666: '`anonymous-namespace'::pythonic::types::dict<`anonymous-namespace'::pythonic::types::str,`anonymous-namespace'::pythonic::types::str>::operator []': 5 overloads have similar conversions
D:\tmp\xalgoenv\lib\site-packages\pythran-0.8.6.dev0+x-py3.6.egg/pythran\pythonic\include\types\dict.hpp(178): note: could be 'const `anonymous-namespace'::pythonic::types::str &`anonymous-namespace'::pythonic::types::dict<`anonymous-namespace'::pythonic::types::str,`anonymous-namespace'::pythonic::types::str>::operator [](const K &) const'
        with
        [
            K=`anonymous-namespace'::pythonic::types::str
        ]
D:\tmp\xalgoenv\lib\site-packages\pythran-0.8.6.dev0+x-py3.6.egg/pythran\pythonic\include\types\dict.hpp(172): note: or       '`anonymous-namespace'::pythonic::types::str &`anonymous-namespace'::pythonic::types::dict<`anonymous-namespace'::pythonic::types::str,`anonymous-namespace'::pythonic::types::str>::operator [](const K &)'
        with
        [
            K=`anonymous-namespace'::pythonic::types::str
        ]
D:\tmp\xalgoenv\lib\site-packages\pythran-0.8.6.dev0+x-py3.6.egg/pythran\pythonic\include\types\dict.hpp(180): note: or       'const V &`anonymous-namespace'::pythonic::types::dict<`anonymous-namespace'::pythonic::types::str,V>::operator []<char[9]>(const OtherKey (&)) const'
        with
        [
            V=`anonymous-namespace'::pythonic::types::str,
            OtherKey=char [9]
        ]
D:\tmp\xalgoenv\lib\site-packages\pythran-0.8.6.dev0+x-py3.6.egg/pythran\pythonic\include\types\dict.hpp(174): note: or       'V &`anonymous-namespace'::pythonic::types::dict<`anonymous-namespace'::pythonic::types::str,V>::operator []<char[9]>(const OtherKey (&))'
        with
        [
            V=`anonymous-namespace'::pythonic::types::str,
            OtherKey=char [9]
        ]
xalgo/_comp.cpp(91): note: or       'built-in C++ operator[(__int64, char [9])'
xalgo/_comp.cpp(91): note: while trying to match the argument list '(`anonymous-namespace'::pythonic::types::dict<`anonymous-namespace'::pythonic::types::str,`anonymous-namespace'::pythonic::types::str>, char [9])'